app/actors/SessionsScheduler.scala

```diff
@@ -233,8 +233,11 @@ class SessionsScheduler @Inject()(sessionsRepository: SessionsRepository,
 
     scheduledEmails = scheduledEmails - key
 
-    emailManager ! EmailActor.SendEmail(
-      emails, fromEmail, "Feedback reminder", views.html.emails.reminder(emailInfo, feedbackUrl).toString)
+    sessions.foreach{ session =>
+      val emailsExceptPresenter = emails.filter(email=> email!=session.email)
```

Comment:
Please make sure code is properly re-formatted before sending pull requests. Using Intellij reformatting could he helpful here.